src/controllers/playback/video/index.js

```diff
@@ -1252,7 +1252,7 @@ export default function (view) {
 
         switch (key) {
             case 'Enter':
-                showOsd();
+                showOsd(e.target.classList.contains('skip-button') ? btnPlayPause : undefined);
```

Comment:
I prefer this to be fixed for the general case (for example, if we add another button). Did you try what I [posted](https://github.com/jellyfin/jellyfin-web/pull/6295#issuecomment-2564754517)?

Reply:
Yes, I tried your suggestion, but it essentially prevented the skip button from gaining focus when the OSD was visible.

Reply:
Would it be better to change this to:

```javascript
showOsd(osdBottomElement.contains(e.target) ? undefined : btnPlayPause);
```

Reply:
> Yes, I tried your suggestion, but it essentially prevented the skip button from gaining focus when the OSD was visible.

It shouldn't gain focus when the OSD is visible. More precisely, it shouldn't grab focus from a valid element that the user interacts with.

We are trying to fix the loss of focus after clicking the skip button, no?
There may be a problem with my suggestion if `showOsd` is called before the focus is blurred.


> Would it be better to change this to:
> 
> ```js
> showOsd(osdBottomElement.contains(e.target) ? undefined : btnPlayPause);
> ```

I guess it will interfere with the subtitle offset dialog - it doesn't belong to `osdBottomElement`.

Reply:
> It shouldn't gain focus when the OSD is visible. More precisely, it shouldn't grab focus from a valid element that the user interacts with.

I mean it's impossible to use the navigation keys to reach the skip button. The focus keeps jumping back to the pause button every time.